data/ka'het missions.txt

```diff
@@ -91,42 +91,111 @@ mission "First Contact: Ka'het: Remnant 1B"
+			`	However unexpected a response like this from the Remnant is, they simply leave your ship shortly after, claiming that it is useless to gather debris like that.`
+				decline
+
+mission "Ringworld Debris: Hai"
+	invisible
+	landing
+	source
+		government "Hai"
+	to offer
+		has "ringworld debris"
+		has "Graveyard Label: offered"
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`Do you want to show the Hai the piece of debris you found in the Graveyard, and ask them what they know about it?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	You ask around for a bit, and eventually are lead to a building where some Hai engineers and shipwrights are gathered, a workshop of some kind.`
+			`	"Greetings, human friend! How may we help you?"`
+			`	You show them the piece of debris, and they observe it for a bit, plugging it like you did on your own ship, and seeing how it behaves. They look puzzled along the entire process, and eventually conclude they don't have the necessary equipment to fully analyze it.`
+			`	"Sorry, whatever it is that you've found, it is too advanced a technology for our systems," they say. You thank them for their time, and head back to your ship.`
+				decline
 
-mission "First Contact: Z Ka'het: Quarg 2"
+mission "Ringworld Debris: Coalition"
+	invisible
 	landing
+	source
+		government "Coalition" "Heliarch"
+	to offer
+		has "ringworld debris"
+		has "Graveyard Label: offered"
+		has "Coalition: First Contact: done"
+	on offer
+		conversation
+			`Do you want to show the members of the Coalition the piece of debris you found in the Graveyard, and ask them what they know about it?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	After asking around, you're directed to a Heliarch building, an outpost. "Of assistance, how may we be, kind visitor?" An agent asks.`
+			`	You show them the dark debris piece, and explain where you've found it. They are clearly surprised by it, but don't hold you up for long, and don't attempt to perform any tests on it.`
+			`	"A piece of Quarg technology, this !" They say, asking again for you to explain where exactly you found it, as they make sure to write it all down this time.`
+			`	They then say that they have experienced with technology they managed to steal from the Quarg during their war, and that the debris is probably a somewhat common component in Quarg equipment.`
+			`	You wonder for a bit if they will ask to buy the piece, judging by how excited they seem, but they finish saying that they have enough technology to study as is, and also that the piece you found has deteriorated much compared to what they have acquired for themselves.`
+			`	You thank them for their time and the explanation, and head back to your ship.`
+				decline
+
+mission "Ringworld Debris: Quarg"
 	invisible
+	landing
+	deadline 1
 	source
-		attributes "quarg"
+		government "Quarg"
```

Comment:
The reason `attributes` was used here is because of Humanika (which has no permanent Quarg population).

Reply:
it having no permanent Quargos is all the more reason to restrict the source to the Quarg government.